app/models/accounting/quickbooks/fetcher.rb

```diff
@@ -0,0 +1,53 @@
+module Accounting
+  module Quickbooks
+    class Fetcher
```

Comment:
Nice class name! Can we please get a general comment describing what the class does.